packages/next-plugin-sentry/README.md

```diff
@@ -0,0 +1,17 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry plugin for Next.js
+
+## Links
+
+- [Official SDK Docs](https://docs.sentry.io/quickstart/)
```

Comment:
```suggestion
- [Official SDK Docs](https://docs.sentry.io/platforms/javascript/guides/nextjs/)
```

We point to the specific SDK docs in similar packages like `vue`, `react`, and `angular`.